base/exports.jl

```diff
@@ -500,88 +499,88 @@ export
     searchsortedlast,
     startswith,
 
-# linear algebra
-    bkfact!,
-    bkfact,
-    chol,
-    cholfact!,
-    cholfact,
-    cond,
-    condskeel,
-    cross,
-    adjoint!,
+# # linear algebra
+#     bkfact!,
+#     bkfact,
+#     chol,
+#     cholfact!,
+#     cholfact,
+#     cond,
+#     condskeel,
+#     cross,
+#     adjoint!,
     adjoint,
-    det,
-    diag,
-    diagind,
-    diagm,
-    diff,
-    dot,
-    eig,
-    eigfact!,
-    eigfact,
-    eigmax,
-    eigmin,
-    eigvals,
-    eigvals!,
-    eigvecs,
-    factorize,
-    givens,
-    hessfact!,
-    hessfact,
-    isdiag,
-    ishermitian,
-    isposdef!,
-    isposdef,
-    issymmetric,
-    istril,
-    istriu,
+#     det,
+#     diag,
+#     diagind,
+#     diagm,
+#     diff,
+#     dot,
+#     eig,
+#     eigfact!,
+#     eigfact,
+#     eigmax,
+#     eigmin,
+#     eigvals,
+#     eigvals!,
+#     eigvecs,
+#     factorize,
+#     givens,
+#     hessfact!,
+#     hessfact,
+#     isdiag,
+#     ishermitian,
+#     isposdef!,
+#     isposdef,
+#     issymmetric,
+#     istril,
+#     istriu,
     kron,
-    ldltfact,
-    ldltfact!,
-    linreg,
-    logabsdet,
-    logdet,
-    lu,
-    lufact!,
-    lufact,
-    lyap,
-    norm,
-    normalize,
-    normalize!,
-    nullspace,
-    ordschur!,
-    ordschur,
-    peakflops,
-    pinv,
-    qr,
-    qrfact!,
-    qrfact,
-    lq,
-    lqfact!,
-    lqfact,
-    rank,
-    scale!,
-    schur,
-    schurfact!,
-    schurfact,
-    svd,
-    svdfact!,
-    svdfact,
-    svdvals!,
-    svdvals,
-    sylvester,
-    trace,
-    transpose!,
+    # ldltfact,
+#     ldltfact!,
+#     linreg,
+#     logabsdet,
+#     logdet,
+#     lu,
+#     lufact!,
+#     lufact,
+#     lyap,
+#     norm,
+#     normalize,
+#     normalize!,
+#     nullspace,
+#     ordschur!,
+#     ordschur,
+#     peakflops,
+#     pinv,
+#     qr,
+#     qrfact!,
+#     qrfact,
+#     lq,
+#     lqfact!,
+#     lqfact,
+#     rank,
+#     scale!,
+#     schur,
+#     schurfact!,
+#     schurfact,
+#     svd,
+#     svdfact!,
+#     svdfact,
+#     svdvals!,
+#     svdvals,
+#     sylvester,
+#     trace,
+#     transpose!,
     transpose,
-    tril!,
-    tril,
-    triu!,
-    triu,
-    vecdot,
-    vecnorm,
-    ⋅,
-    ×,
+#     tril!,
+#     tril,
+#     triu!,
+#     triu,
+#     vecdot,
+#     vecnorm,
+#     ⋅,
+#     ×,
```

Comment:
All these should just be deleted.

Reply:
Part of the working process, will clean up the PR :)